client/src/lib/CaptchaV2.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+  import {createEventDispatcher} from "svelte";
+  export let captchaKey: string;
+
+  const dispatch = createEventDispatcher<{ token: string }>();
+
+  const captchaId = 'captcha_element';
+
+  window.captchaLoaded = () => {
```

Comment:
I have 0 experience with Svelte. Is it normal to add things globally to `window`? Seems to me like something to avoid.

Reply:
Not really, but the captcha api expect windows methods, so I don't have any other way of exposing a method for it to call.

If I would be able to pass a reference to a method I would, but unfortunately it doesn't work :(